config/secrets.yml.dev

```diff
@@ -97,7 +97,7 @@ dev_settings: &DEV_SETTINGS
     tos_url: https://search.gov/tos
     page_not_found_url: https://www.usa.gov/search-error
     admin_email_address: usasearchoutbound@search.gov
-    support_email_address: search@support.digitalgov.gov
```

Comment:
Per our discussion, I'll put in a sub-task to make this change in the cookbooks, as this file isn't used in production.

Reply:
See: https://github.com/GSA/usasearch-cookbooks/pull/601